### ISSUE TITLE
Project Detail Edit: Hide info that needs approval from PM view

### ISSUE THREAD

**Issue by maburke09** [hackforla/VRMS#701]
### Overview
As a Org rep, I need to have pms be able to only change things themselves on their project object that do not require approval from Org so that we can reduce the amount of time spent on editing of database on their behalf

### Details
Additional options will be available to admin level and in future the pms will be able to change those items through a review process that will exist in 0.5. Also, we want them to be able to see all their info, with a note saying something about reach out to the #admin slack channel to change grayed out items.

### Action Items
Review list of items and put a check box by anything they should be able to change.

### Resources/Instructions
List of items that can be edited for bonnie to write a requirements doc for hiding

- [ ] Name: 
- [ ] Description: 
- [ ] GitHub URL: 
- [ ] Slack URL: 
- [x] Google Drive URL:
- [ ] HfLA Website URL: 
- [ ] Domain Name URL: 
- [x] Video Conference Link: 
- [x] Partners (comma separated): 
- [x] Times for the meetings
- [ ] Location: remote

**Comment by drubgrubby** [comment]
per conversation with Bonnie. 
Add fields to db:
Demo site link
Test site link

Bonnie says users can edit those fields.

**Comment by chukalicious** [comment]
@drubgrubby Hi David! From what I understood from our previous conversation users with an access level of "user" can only edit the fields checked on the list above, users with an access level of admin can change every field of the project. PR draft https://github.com/hackforla/VRMS/pull/794 addresses that. 

FYI: Before these changes were implemented and I logged in with my user access level account, I could make any change to the project object, now they are restricted by "user".

Let's take a look when you have a chance and make sure the desired effect has been achieved. I want to make sure I understood the problem. 

Thank you!
